index.html

```diff
@@ -1,111 +1,424 @@
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+        </div>
+      </section>
+      <section class="faqs">
+        <div>
+          <h1>FAQs</h1>
+        </div>
+        <div>
+          <div>
+            <h1>What are the NTFs?</h1>
+            <button>+</button>
+          </div>
+          <p>
+            Urna vitae erat et lacus, consectetur ac nulla vestibulum lobortic.
+            Nula dapibus urna volutpat vestibulum lobortic. Nulla dapibus urna
+            volutpat venenatis, risus faucibus.
+          </p>
+          <div>
+            <h1>How do i get NTFs?</h1>
+            <button>+</button>
+          </div>
+          <div>
+            <h1>How can we buy your NTFs?</h1>
+            <button>+</button>
+          </div>
+          <div>
+            <h1>Who are the team behind the project?</h1>
+            <button>+</button>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div>
+        <div class="logo">
+          <a class="logo" href="#">Binabox</a>
+          <p>We are the best way to check the rarity of NFT collection.</p>
+        </div>
+        <div class="menu">
+          <ul>
```

Comment:
The 'ul' tag contains only list items. It can not contain any other tags except the 'li' tag